src/pizzeria/Ingredients.java

```diff
@@ -0,0 +1,78 @@
+package pizzeria;
+
+public enum Ingredients {
```

Comment:
Your code works, but let's change the enum Ingredients like this url
[https://www.tutorialspoint.com/can-we-create-an-enum-with-custom-values-in-java]

Reply:
done